packages/react/src/ActionList/ActionList.features.stories.tsx

```diff
@@ -440,10 +442,10 @@ export const InactiveItem = () => {
 
 export const Links = () => (
   <>
-    <Heading as="h1" id="list-heading" sx={{fontSize: 1}}>
```

Comment:
related to ActionList.Heading sub component https://github.com/primer/react/pull/4390/files#r1525664183